spec/amber/support/file_encryptor.cr

```diff
@@ -0,0 +1,52 @@
+require "../../../spec_helper"
+
+describe Amber::Support::FileEncryptor do
+  Dir.mkdir_p("./tmp")
+  context "#global_encryption_key" do
```

Comment:
Had a quick chat with @elorest on gitter and I was confused about the amber secret_key this key does not correspond to the secret_key_base found in the YAML this key is only used for encryption and decryption of the yaml files.

Reply:
You can rename `global_encryption_key` to just `encryption_key` since there is only one and only one encryption key. Global makes it feel that there could be another secret key somewhere else.

Reply:
Done.